app/webpack/stylesheets/components/_custom_styles.scss

```diff
@@ -1,3 +1,17 @@
 .vcd-capitalize-first-letter:first-letter {
   text-transform: uppercase;
 }
+
+// Display for js only
+body:not(.js-enabled) .app-js-only {
+  display: none;
+}
+
+// Display for no-js only
+.js-enabled .app-no-js-only {
+  display: none;
+}
```

Comment:
Is this code block still required.

Reply:
I think so.  We still should not be asking the question to people without JS.

Reply:
We've implemented a slightly different banner and form for where JS is not enabled, so the form will not be rendered if the user has disabled JS